variables.tf

```diff
@@ -7,7 +7,23 @@ variable "ram_resource_share_enabled" {
 variable "ram_principal" {
   type        = string
   default     = null
-  description = "The principal to associate with the resource share. Possible values are an AWS account ID, an Organization ARN, or an Organization Unit ARN. If this is not provided and `ram_resource_share_enabled` is set to `true`, the Organization ARN will be used"
+  description = "DEPRECATED, please use ram_principals instead."
```

Comment:
maybe add the old description after the deprecation message?
@alexjurkiewicz

Reply:
done, thanks!